### PR TITLE
testing/sblim-wbemcli: new aport

### PR DESCRIPTION
**Pull request by goricz**
https://sourceforge.net/projects/sblim/files/sblim-wbemcli
WBEM Command Line Interface for CIMOM access

### CODE REVIEW

File: testing/sblim-wbemcli/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributor: Jirka Dutka <jirka@dutka.net>
+# Maintainer: Jirka Dutka <jirka@dutka.net>
+pkgname=sblim-wbemcli
+pkgver=1.6.3
+pkgrel=0
+pkgdesc="WBEM Command Line Interface for CIMOM access"
+url="https://sourceforge.net/projects/sblim/files/sblim-wbemcli"
+arch="all"
+license="CPL-1.0,EPL-1.0"
```
Comment (maxice8) [inline testing/sblim-wbemcli/APKBUILD]: the license is separated using a SPDX identifier.

AND -> When the project is under both licenses
OR -> If the user can pick under which license to use

File: testing/sblim-wbemcli/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+depends="libcurl"
```
Comment (maxice8) [inline testing/sblim-wbemcli/APKBUILD]: I believe this is found automatically via our so: providers/depends system

File: testing/sblim-wbemcli/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+makedepends="curl-dev"
+subpackages="$pkgname-doc"
+source="https://downloads.sourceforge.net/project/sblim/$pkgname/$pkgver/$pkgname-$pkgver.tar.bz2"
+options="!check"
+# builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/sblim-wbemcli/APKBUILD]: this can be removed

File: testing/sblim-wbemcli/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+pkgrel=0
+pkgdesc="WBEM Command Line Interface for CIMOM access"
+url="https://sourceforge.net/projects/sblim/files/sblim-wbemcli"
+arch="all"
+license="CPL-1.0,EPL-1.0"
+depends="libcurl"
+makedepends="curl-dev"
+subpackages="$pkgname-doc"
+source="https://downloads.sourceforge.net/project/sblim/$pkgname/$pkgver/$pkgname-$pkgver.tar.bz2"
+options="!check"
```
Comment (maxice8) [inline testing/sblim-wbemcli/APKBUILD]: Please add a comment on why this is not enabled

**Comment by Cogitri** [comment]
Seems like this doesn't build on aarch64:

`configure: error: cannot guess build type; you must specify one`

This can be fixed with `update_config_sub`, see other packages for how to do this.

**Comment by goricz** [comment]
> Seems like this doesn't build on aarch64:
> 
> `configure: error: cannot guess build type; you must specify one`
> This can be fixed with `update_config_sub`, see other packages for how to do this.

Hi, 

tried to add prepare section with update_config_sub, no luck, still failing for aarch64. 
Can I just exclude this architecture?

**Comment by Cogitri** [comment]
Seems to work now, good job :)

Since my laptop is in repair (*sigh*, again) @maxice8 can you please take another look & merge?

**Pull request closed without merging**